tutorials/managing-gcp-projects-with-terraform/index.md

```diff
@@ -0,0 +1,285 @@
+---
+title: Managing GCP Projects with Terraform
+description: Learn how to manage projects within an organization and project resources with Terraform.
+author: danisla
+tags: Terraform
+date_published: --
+---
+
+This tutorial demonstrates how to create and manage projects on Google Cloud Platform with Terraform. With Terraform, many of your resources such as projects, IAM policies, networks, Compute Engine instances, and Container Engine clusters can be managed, versioned, and easily recreated for your organization or teams. The state that Terraform generates is saved to Google Cloud Storage for persistence.
+
+Incremental changes made from the cloud console or gcloud command-line tool are difficult to track and recreate if needed later down the road. With Terraform, many of your resources like Projects, IAM policies, Networks, Compute Instances, and Container Engine Clusters can be managed and versioned and easily recreated for your organization or teams. The state that Terraform generates is saved to Google Cloud Storage for persistence.
+
+## Objectives
+
+- Create a Terraform Admin Project for the service account and remote state bucket.
+- Grant Organization-level permissions to the service account.
+- Configure the remote state in GCS.
+- Use Terraform to provision a new project and an instance in that project.
+ 
+Architecture diagram for tutorial components:
+
+**Figure 1.** *Architecture diagram for tutorial components*
+![architecture diagram](./diagram.png)
```

Comment:
Change `./diagram.png` to `https://storage.googleapis.com/gcp-community/tutorials/managing-gcp-projects-with-terraform/diagram.png`